PWGHF/Tasks/taskPidStudies.cxx

```diff
@@ -49,8 +50,8 @@ DECLARE_SOA_COLUMN(MassAntiLambda, massAntiLambda, float); //! Candidate mass
 DECLARE_SOA_COLUMN(Pt, pt, float);                         //! Transverse momentum of the candidate (GeV/c)
 DECLARE_SOA_COLUMN(PtPos, ptPos, float);                   //! Transverse momentum of positive track (GeV/c)
 DECLARE_SOA_COLUMN(PtNeg, ptNeg, float);                   //! Transverse momentum of negative track (GeV/c)
-DECLARE_SOA_COLUMN(TpcInnerParPos, tpcInnerParPos, float); //! Momentum of positive track at inner wall of TPC (GeV/c)
-DECLARE_SOA_COLUMN(TpcInnerParNeg, tpcInnerParNeg, float); //! Momentum of negative track at inner wall of TPC (GeV/c)
+DECLARE_SOA_COLUMN(TpcPtPos, tpcPtPos, float);             //! Transverse Momentum of positive track at inner wall of TPC (GeV/c)
+DECLARE_SOA_COLUMN(TpcPtNeg, tpcPtNeg, float);             //! Transverse Momentum of negative track at inner wall of TPC (GeV/c)
```

Comment:
What is the reason for replacing the quantities and columns?

Reply:
Dear Vit, we decided to change the quantity stored in the tables because we are interested in a comparison of our results with the standard pt and the TPC pt. If we do not compute it on the fly, we would need to add two more columns in the V0 table and one in the cascades table to store the eta of the daughter tracks.

Reply:
OK, thanks for the clarification.
Then please rename the new columns to match the names of other pT columns.